### DISCUSSION TITLE
Cache statistic

### DISCUSSION THREAD

**Discussion by VolodymyrBor** [Yiling-J/theine#16] [Q&A] [answered]
Is there any way to view cache static, such as hits, misses, maxsize, and currsize? Just like in [lru_cache](https://docs.python.org/3/library/functools.html#functools.lru_cache)

It would be useful to choose the right Cache parameters based on such statistics.

**Comment by Yiling-J** [comment, marked as answer]
@VolodymyrBor Currently Theine does not provide built-in statistics, I create an issue https://github.com/Yiling-J/theine/issues/17 and will start work on it this weekends

**Reply by VolodymyrBor** [reply]
Thanks, this will be a useful feature

**Reply by Yiling-J** [reply]
@VolodymyrBor 0.4.1 released, please try new stats API. To get maxsize, you can use cache.max_size, to get currsize, you can use len(cache)
